### DISCUSSION TITLE
Is it possible to change the generated style.css imports order?

### DISCUSSION THREAD

**Discussion by anubra266** [chakra-ui/panda#901] [Q&A]
### Discussed in https://github.com/chakra-ui/panda/discussions/899

<div type='discussions-op-text'>

<sup>Originally posted by **TheOrdinaryDev** July  1, 2023</sup>
With `preflight: true` and `staticCss` config options generated `style.css` file looks like this
```css
@layer reset, base, tokens, recipes, utilities;

@import './global.css';

@import './static.css';

@import './reset.css';

@import './tokens/index.css';

@import './tokens/keyframes.css';
```
`reset.css` is after `static.css`, so resetting styles overrides static styles.

Is it possible to change imports order in generated `style.css` file?</div>

**Comment by astahmer** [comment]
duplicate of https://github.com/chakra-ui/panda/discussions/899
